doc/source/tutorial/tutorial_IO.ipyml

```diff
@@ -201,28 +219,56 @@ cells:
     # a  | b  | data  |
     # a0 | b0 | 0     |
     # a0 | b1 | 1     |
-    # a1 | b0 | 2     |
-    # a1 | b1 | 3     |
+    # a0 | b2 | 2     |
+    # a1 | b0 | 3     |
+    # a1 | b1 | 4     |
+    # a1 | b2 | 5     |
 
-  id: 8
+  id: 9
 
 - markdown: |
-    ##### Exporting only data
+    ##### Exporting only data (without headers)
+    
+    To export only data in Excel sheets, you can choose between setting the ``header`` argument of the ``dump`` method to False or avoid calling ``dump``. 
+    
+    This can be usefull as you to export data following a specific format. For example, let's consider you have a 2D array `pop` representing the projection of the population of some selected cities until 2025. Let's assume this array has two axes `city` and `time` respectively but you need to include the associated continent in the output file. What you want is:  
     
-    To export only data in Excel sheets, you can set argument ``header`` of the method ``dump`` to false or not calling ``dump``:
+    |city     |	continent	  | 2015 | 2016 | 2017 | 2018 | 2019 | 2020 | 2021 | 2022 | 2023 | 2024 | 2025 |
+    | ------- | ------------- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- |
+    |Brussel  |	Europe	      | 0    | 1	| 2	   | 3	  | 4    | 5    | 6    | 7    | 8    | 9    | 10   |
+    |Paris	  | Europe	      | 11   | 12	| 13   | 14	  | 15   | 16	| 17   | 18   | 19   | 20   | 21   |
+    |New York |	North America |	22   | 23	| 24   | 25	  | 26   | 27	| 28   | 29   | 30   | 31   | 32   |
+    |Mexico	  | North America |	33   | 34	| 35   | 36	  | 37   | 38	| 39   | 40   | 41   | 42   | 43   |
+    |Cairo	  | Africa	      | 44   | 45	| 46   | 47	  | 48   | 49	| 50   | 51   | 52   | 53   | 54   |
+    |Tokyo	  | Asia	      | 55   | 56	| 57   | 58	  | 59   | 60	| 61   | 62   | 63   | 64   | 65   |
+    |Moscow	  | Asia	      | 66   | 67	| 68   | 69	  | 70   | 71	| 72   | 73   | 74   | 75   | 76   |
+    
+    To get this output result, one way is to dump axes (city, continent and time) and data separately as describe below:
 
 
 - code: |
-    arr = ndtest((2, 2))
+    city = Axis("city=Brussel,Paris,New York,Mexico,Cairo,Tokyo,Moscow")
+    continent = Axis("continent=Europe,Europe,North America,North America,Africa,Asia,Asia")
+    time = Axis("time=2015..2025")
+    pop = ndtest((city, time))
     
-    with open_excel('new_excel_file.xlsx') as wb:
-        # export only data
-        wb['data_only'] = arr.dump(header=False)
+    with open_excel('pop_projection.xlsx', overwrite_file=True) as wb:
+        # create new sheet 'pop_2015_2025'
+        wb['pop_2015_2025'] = []
+        sh = wb['pop_2015_2025']
```

Comment:
Add a comment for this line too (like in the previous example)? It doesn't hurt to repeat some stuff :)